bundles/org.openhab.automation.jsscripting/README.md

```diff
@@ -30,7 +30,7 @@ to common openHAB functionality within rules including items, things, actions, l
 
 This add-on includes by default the [openhab-js](https://github.com/openhab/openhab-js/) NPM library and exports it's namespaces onto the global namespace. This allows the use of `items`, `actions`, `cache` and other objects without the need to explicitly import using `require()`.  This functionality can be disabled for users who prefer to manage their own imports via the add-on configuration options. 
 
-![OpenHAB Rule Configuration](./docs/settings.png)
+![OpenHAB Rule Configuration](./doc/settings.png)
```

Comment:
Can you also replace all the `OpenHAB` occurrences with `openHAB` ? :slightly_smiling_face: